core/domain/classifier_services.py

```diff
@@ -124,6 +124,57 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def train(exploration):
```

Comment:
Sorry, I don't understand. Isn't the ultimate VM approach also async? I.e. when the creator saves they aren't waiting for the VM to respond before they can make another request, right?